app/src/androidTest/java/net/skyscanner/backpack/icon/BpkIconTest.kt

```diff
@@ -0,0 +1,87 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.icon
+
+import android.content.res.Resources
+import android.graphics.drawable.Drawable
+import android.graphics.drawable.VectorDrawable
+import android.view.ViewGroup
+import android.widget.GridLayout
+import android.widget.ImageView
+import androidx.appcompat.content.res.AppCompatResources
+import net.skyscanner.backpack.BpkSnapshotTest
+import net.skyscanner.backpack.BpkTestVariant
+import net.skyscanner.backpack.R
+import org.junit.Assume.assumeTrue
+import org.junit.Before
+import org.junit.Test
+
+class BpkIconTest : BpkSnapshotTest() {
+
+  @Before
+  fun setup() {
+    setDimensions(600, 300)
+  }
+
+  @Test
+  fun testAllSmallIcons() {
+    assumeTrue(BpkTestVariant.current == BpkTestVariant.Default)
```

Comment:
this runs it only on the default variant as theming doesn't affect the raw icons